service/configure.ac

```diff
@@ -74,6 +74,20 @@ fi
 [enable_coverage="0"]
 )
 
+AC_ARG_ENABLE([docs],
+  [AS_HELP_STRING([--disable-docs], [Disable the use of the Python sphinx tool to generate documentation])],
```

Comment:
In the latest release (https://github.com/geopm/geopm/blob/v1.1.0/configure.ac#L274), this was `--disable-doc`.

I was going to recommend keeping the old name or adding an alias to avoid breaking any existing build scripts that someone might have. But on second thought, I think that doesn't matter since the build process is different for the next release.

Reply:
Thanks for pointing that out.  The build target is now "make docs" so I think this should be clearer to users who are not familiar with our old options...however, I can see the argument to keep it the same as it used to be.